go.mod

```diff
@@ -493,3 +493,5 @@ replace github.com/stretchr/testify => github.com/stretchr/testify v1.8.4
 //
 // Then, run `make modtidy-all` in this repository.
 // This ensures that go.mod and go.sum are up-to-date for each go.mod file.
+
+replace github.com/diagridio/go-etcd-cron => github.com/joshvanl/go-etcd-cron v0.0.0-20240724013448-56db9d13f905
```

Comment:
This should be changed from your fork yeah?

Reply:
Yes, just need this first https://github.com/diagridio/go-etcd-cron/pull/33

Reply:
Merged. Released v0.2.2 now.